modules/apps/headless/headless-space/headless-space-test/src/testIntegration/java/com/liferay/headless/space/resource/v1_0/test/SpaceResourceTest.java

```diff
@@ -0,0 +1,171 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.space.resource.v1_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.depot.model.DepotEntry;
+import com.liferay.depot.service.DepotEntryGroupRelLocalService;
+import com.liferay.depot.service.DepotEntryLocalService;
+import com.liferay.headless.space.client.dto.v1_0.Space;
+import com.liferay.headless.space.client.problem.Problem;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.test.rule.Inject;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Roberto Díaz
+ */
+@RunWith(Arquillian.class)
+public class SpaceResourceTest extends BaseSpaceResourceTestCase {
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+	}
+
+	@After
+	@Override
+	public void tearDown() throws Exception {
+		super.tearDown();
+	}
+
+	@Override
+	@Test
+	public void testDeleteSpace() throws Exception {
+		super.testDeleteSpace();
+
+		// Nonexistent space ID
+
+		long siteId = RandomTestUtil.randomLong();
+
+		try {
+			spaceResource.deleteSpace(siteId);
+
+			Assert.fail();
+		}
+		catch (Problem.ProblemException problemException) {
+			Problem problem = problemException.getProblem();
+
+			Assert.assertEquals("NOT_FOUND", problem.getStatus());
+			Assert.assertNull(problem.getTitle());
+		}
+	}
+
+	@Override
+	@Test
+	public void testDeleteSpaceLinkToSite() throws Exception {
+		Space space = testDeleteSpaceLinkToSite_addSpace();
+
+		_assertLinkedSites(space);
+
+		space = spaceResource.deleteSpaceLinkToSite(
+			space.getId(), testGroup.getGroupId());
+
+		Assert.assertTrue(ArrayUtil.isEmpty(space.getLinkedSiteIds()));
+		Assert.assertTrue(
+			ArrayUtil.isEmpty(space.getLinkedSitesExternalReferenceCodes()));
+	}
```

Comment:
You need to invoke super.test* for every test you override.

Reply:
I'm not sure, because delete methods expect a 204 error. in this case this delete method is not returning this but the space without the linkToSite

Reply:
There are other cases: for example com.liferay.headless.commerce.admin.inventory.resource.v1_0.test.WarehouseOrderTypeResourceTest#testDeleteWarehouseOrderType doesn't call the super